File: JS/10.js
// Reverse Geometric Series (RECURSION)
function Power(i, expo, num) {
    if (i <= expo) {
        return num * Power(i + 1, expo, num);
    }
    return 1
}

function Geometric(start, range) {
    if (start <= range) {
        console.log(Power(1, start, 3));
        return Geometric(start + 1, range);
    }
    return 0;
}

Geometric(1, 10);

File: C/10.c
// Number pattern (RECURSION)
#include <stdio.h>
int pattern(int i);
int Incre(int num, int j);
int main()
{
    pattern(1);
    return 0;
}
int pattern(int i)
{
    if (i <= 5)
    {
        Incre(1, i);
        printf("\n");
        return pattern(i + 1);
    }
    return 0;
}
int Incre(int num, int j)
{
    if (num <= j)
    {
        printf("%d ", num);
        return Incre(num + 1, j);
    }
    return 0;
}